BeerOverflowWindowsApp/Database/BarsDatabase.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.Entity;
+using BeerOverflowWindowsApp.DataModels;
+
+namespace BeerOverflowWindowsApp
```

Comment:
Namespace should BeerOverflowWindowsApp.Database. Also for everything in this and FileIO folder.

Reply:
roger that